dbms/src/Flash/Coprocessor/DAGQueryBlock.cpp

```diff
@@ -40,6 +40,7 @@ bool isSourceNode(const tipb::Executor * root)
         || root->tp() == tipb::ExecType::TypePartitionTableScan
         || root->tp() == tipb::ExecType::TypeWindow
         || (root->tp() == tipb::ExecType::TypeSort && root->sort().ispartialsort());
+    // ywq test: question: why is_partialsort.
```

Comment:
`partial sort` is only for the window function.
Currently TiFlash does not support distributed sort.